cmd/buildah/run.go

```diff
@@ -85,9 +89,9 @@ func runCmd(c *cli.Context) error {
 		Runtime:  runtime,
 		Args:     flags,
 	}
-	runerr := builder.Run(c.Args(), options)
+	runerr := builder.Run(args, options)
 	if runerr != nil {
-		logrus.Debugf("error running %v in container: %v", c.Args(), runerr)
+		logrus.Debugf("error running %v in container: %v", args, runerr)
```

Comment:
That's true, probably better to have args here as we used it in the Run at line92.  In that case can/should we add the name variable to this debug in case it's helpful?